src/main/java/org/lamisplus/modules/base/service/ApplicationCodesetService.java

```diff
@@ -84,6 +85,8 @@ public ApplicationCodeset update(Long id, ApplicationCodesetDTO applicationCodes
         }
         final ApplicationCodeset applicationCodeset = applicationCodesetMapper.toApplicationCodeset(applicationCodesetDTO);
         applicationCodeset.setId(id);
+        applicationCodeset.setArchived(UN_ARCHIVED);
```

Comment:
Whats the difference between archived and active?

Reply:
When a record is archived, it automatically becomes inactive. I think the active field is somewhat redundant.